identity-service/src/routes/trackListens.js

```diff
@@ -381,190 +379,115 @@ module.exports = function (app) {
+        const { res: solTxSignature, error } =
+          await transactionHandler.handleTransaction({
+            instructions,
+            skipPreflight: false, // TODO
+            feePayerOverride: feePayerAccount,
+            retry: true
           })
-          const feePayerAccount = getFeePayerKeypair(false)
-          let solTxSignature
-          if (sendRawTransaction) {
-            req.logger.debug(
-              `TrackListen tx submission, trackId=${trackId} userId=${userId} - sendRawTransaction`
-            )
-            solTxSignature = await sendAndSignTransaction(
-              connection,
-              trackListenTransaction,
-              feePayerAccount,
-              timeout,
-              logger
-            )
-          } else {
-            await retry(
-              async () => {
-                req.logger.debug(
-                  `TrackListen tx submission, trackId=${trackId} userId=${userId} - sendAndConfirmTransaction`
-                )
-                solTxSignature = await solanaWeb3.sendAndConfirmTransaction(
-                  connection,
-                  trackListenTransaction,
-                  [feePayerAccount],
-                  {
-                    skipPreflight: false,
-                    commitment: config.get('solanaTxCommitmentLevel'),
-                    preflightCommitment: config.get('solanaTxCommitmentLevel')
-                  }
-                )
-              },
-              {
-                // Retry function 3x by default
-                // 1st retry delay = 500ms, 2nd = 1500ms, 3rd...nth retry = 8000 ms (capped)
-                minTimeout: 500,
-                maxTimeout: 8000,
-                factor: 3,
-                retries: 3,
-                onRetry: (err, i) => {
-                  if (err) {
-                    req.logger.error(
-                      `TrackListens tx retry error, trackId=${trackId} userId=${userId} : ${err}`
-                    )
-                  }
-                }
-              }
-            )
-          }
-
-          req.logger.debug(
-            `TrackListen tx confirmed, ${solTxSignature} userId=${userId}, trackId=${trackId}, sendRawTransaction=${sendRawTransaction}`
-          )
 
-          // Increment success tracker
-          await redis.incr(trackingRedisKeys.success)
-          await redis.zadd(
-            TRACKING_LISTEN_SUCCESS_KEY,
-            Date.now(),
-            Date.now() + entropy
-          )
-          return successResponse({
-            solTxSignature
-          })
-        } catch (e) {
+        if (error) {
           return errorResponseServerError(
-            `TrackListens tx error, trackId=${trackId} userId=${userId} : ${e}`
+            `TrackListens tx error, trackId=${trackId} userId=${userId} : ${error}`
           )
         }
-      }
 
-      // TODO: Make all of this conditional based on request parameters
```

Comment:
What was this doing? Do we still need? Oh is this the fallback to non-solana listens?

Reply:
yeah this was the fallback flow that's no longer needed